palm/plugins/dbt/commands/cmd_deps.py

```diff
@@ -0,0 +1,10 @@
+import click
+from palm.plugins.dbt.dbt_palm_utils import dbt_env_vars
+
+
+@click.command('deps')
+@click.pass_context
+def cli(ctx):
```

Comment:
Why no pass_obj and environment here?

Reply:
I did it this way for consistency with the other commands in this repo. I think we'll update all of them in one go :)